src/main/java/seedu/address/logic/commands/AddMemberCommand.java

```diff
@@ -14,34 +11,28 @@
 /**
  * Adds a person to the address book.
  */
-public class AddCommand extends Command {
+public class AddMemberCommand extends Command {
 
     public static final String COMMAND_WORD = "add";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
```

Comment:
```suggestion
    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a member to the member list. "
```
Maybe we can write add a member or add a person to the member list?